src/V3Delayed.cpp

```diff
@@ -474,7 +474,9 @@ class DelayedVisitor final : public VNVisitor {
         m_inSuspendableOrFork = true;
         iterateChildren(nodep);
     }
-    void visit(AstCAwait* nodep) override { m_timingDomains.insert(nodep->sensesp()); }
+    void visit(AstCAwait* nodep) override {
+        if (nodep->sensesp()) { m_timingDomains.insert(nodep->sensesp()); };
```

Comment:
```suggestion
        if (nodep->sensesp()) m_timingDomains.insert(nodep->sensesp());
```
I think that's the convention in Verilator? Either way, no `;` after `if`'s curly braces.

Reply:
Right. Fixed.